.github/workflows/reusable-validate-app.yml

```
@@ -33,16 +29,20 @@ jobs:
 
       - name: Run tests
         id: run_tests
+        env:
+          TESTS_REQUIRES_DATABASE: ${{ inputs.requires-database }}
```

actionlint (1.7.7)

property "requires-database" is not defined in object type {app: string}

(expression)